scout/parse/variant/genotype.py

```diff
@@ -111,6 +141,47 @@ def parse_genotype(variant, ind, pos):
     return gt_call
 
 
+def get_mei_reads(variant: cyvcf2.Variant, pos: Dict[str, int]) -> Tuple[int, ...]:
+    """Get MEI caller read details from FORMAT tags.
+    Returns:
```

Comment:
This Returns can be removed, since you already added a return type to the function above!

Reply:
Ok!